core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -40,20 +49,20 @@ export interface ExplanationBackendDict {
 
 export interface ISolutionBackendDict {
   'answer_is_exclusive': boolean;
-  'correct_answer': string;
+  'correct_answer': IInteractionAnswer;
   'explanation': ExplanationBackendDict;
 }
 
 export class Solution {
   ehfs: ExplorationHtmlFormatterService;
   shof: SubtitledHtmlObjectFactory;
   answerIsExclusive: boolean;
-  correctAnswer: any;
+  correctAnswer: IInteractionAnswer;
   explanation: SubtitledHtml;
   constructor(
       ehfs: ExplorationHtmlFormatterService,
       shof: SubtitledHtmlObjectFactory,
-      answerisexclusive: boolean, correctanswer: any,
+      answerisexclusive: boolean, correctanswer: IInteractionAnswer,
```

Comment:
Please fix the names of these parameters to use `camelCase`.

Reply:
done